src/main/resources/com/microfocus/application/automation/tools/results/parser/nunit3/nunit-to-junit.xsl

```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0"
+	xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+	<xsl:output method="xml" indent="yes" cdata-section-elements="system-out system-err" />
+
+	<!-- NUnit2 results format -->
+	<xsl:template match="/test-results">
+	<testsuites>
+		<xsl:for-each select="test-suite//results//test-case[1]">
+
+			<xsl:for-each select="../..">
+				<xsl:variable name="firstTestName"
+					select="results/test-case[1]/@name" />
+					<xsl:variable name="testFixtureName">
+					<xsl:choose>
+						<!-- we have a classic method name -->
+						<xsl:when test="substring($firstTestName, string-length($firstTestName)) = ')'">
+							<xsl:value-of select="substring-before($firstTestName, concat('.', @name))"></xsl:value-of>
+						</xsl:when>
+
+						<!-- we have either a custom name, or a test name -->
+						<xsl:otherwise>
+							<xsl:variable name="testMethodName">
+								<xsl:call-template name="lastIndexOf">
+									<xsl:with-param name="string" select="$firstTestName" />
+									<xsl:with-param name="char"  select="'.'" />
+								</xsl:call-template>
+							</xsl:variable>
+
+							<xsl:choose>
+								<!-- If we didn't find any dot, it means we have just the test name -->
+								<xsl:when test="$testMethodName=$firstTestName">
+									<xsl:value-of select="concat(substring-before($firstTestName, @name), @name)" />
+								</xsl:when>
+								<xsl:otherwise>
+									<xsl:value-of select="substring-before($firstTestName, concat('.', $testMethodName))" />
+								</xsl:otherwise>
+							</xsl:choose>
+						</xsl:otherwise>
+					</xsl:choose>
+				</xsl:variable>
+                <!--
+                <xsl:variable name="testFixtureName"
```

Comment:
Remove unnecessary commented out code, in case you want to save it for later open the relevant Jira ticket.